pangeo_forge_runner/commands/bake.py

```diff
@@ -261,7 +262,7 @@ def start(self):
 
                 # if pangeo-forge-recipes is <=0.9, we have to specify a requirements.txt
                 # file even if it isn't present, as the image used otherwise will not have pangeo-forge-recipes
-                if isinstance(recipe, PTransform):
+                if isinstance(recipe, PTransform) or inspect.isfunction(recipe):
```

Comment:
Can we use [callable](https://docs.python.org/3/library/functions.html#callable) instead?

Reply:
That seems totally doable - I wasn't actually familiar with `inspect.isfunction` so perhaps there are implications here that I'm unfamiliar with?

Reply:
Yeah, `callable` will appropriately check for anything with the [callable protocol](https://docs.python.org/3/c-api/call.html) as well as [callable classes](https://docs.python.org/3/reference/datamodel.html#class-instances) while `inspect.isfunction` will explicitly check just for functions. Since the goal here is to just call the recipe (rather than do something specific to functions, like look at their source), I think `callable` is the right thing here.